README.md

```diff
@@ -41,7 +41,7 @@ NOTE: Be aware that the configuration in debian/rules expects tabs instead of sp
 
 Once the package is built, you have a virtualenv contained in a Debian
 package and upon installation it gets placed, by default, under
-`/usr/share/python/<packagename>`.
+`/opt/venvs/<packagename>`.
```

Comment:
I really don't think `/opt` is an appropriate place for Debian packages to be installing applications by default.

I left a more detailed comment: https://github.com/spotify/dh-virtualenv/issues/159#issuecomment-239040540